ui/viewmodel/wallet/assets_view.cpp

```diff
@@ -30,25 +30,34 @@ QAbstractItemModel* AssetsViewModel::getAssets()
     return _assets.get();
 }
 
-int AssetsViewModel::getSelectedAsset() const
+QVector<beam::Asset::ID> AssetsViewModel::getSelectedAssets()
 {
-    return _selectedAsset.is_initialized() ? static_cast<int>(*_selectedAsset) : -1;
+    return _selectedAssets;
+}
+
+void AssetsViewModel::setSelectedAssets(QVector<beam::Asset::ID> assetIds)
+{
+    _selectedAssets = assetIds;
+    _settings.setLastAssetSelection(_selectedAssets);
+    emit selectedAssetChanged();
 }
 
 void AssetsViewModel::setSelectedAsset(int assetId)
```

Comment:
Are we need 2 methods with same names? Can we implement 1 universal method? Or we can rename "void AssetsViewModel::setSelectedAsset(int assetId)" to "addAssetToSelected".